adhocteam/Head-Start-TTADP
# fix aria role in button select component

## Description of change

"Aria-pressed" is used in the ButtonSelect component to denote which button in the button group is selected. However, a UI change sometime in the past few months seems to have broken this functionality. This is a simple fix for that bug.

## How to test
Inspect the button with the checkmark in the region selector (you may have to use the "Break on subtree modification" setting in your dev tools since the blur behavior makes this difficult). The checked button control should be aria-pressed="true". The non-checked buttons should be aria-pressed="false"

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-0


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
